map_launch/package.xml

```diff
@@ -6,10 +6,16 @@
 
   <maintainer email="ryohsuke.mitsudome@tier4.jp">mitsudome-r</maintainer>
 
-  <license>Apache 2</license>
+  <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
+  <exec_depend>map_loader</exec_depend>
+  <exec_depend>map_tf_generator</exec_depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
```

Comment:
@jilaada given that this package does not contain any C++, I'd replace this with `ament_lint_common`, what do you think?

Reply:
Changes should have been made now @esteve!